be/src/storage/task/engine_publish_version_task.cpp

```diff
@@ -146,8 +146,8 @@ OLAPStatus EnginePublishVersionTask::finish() {
         }
     }
 
-    VLOG(1) << "finish to publish version on transaction."
-            << "transaction_id=" << transaction_id << ", error_tablet_size=" << _error_tablet_ids->size();
+    LOG(INFO) << "finish to publish version on transaction."
```

Comment:
Better to keep VLOG, this will generated too many log.

Reply:
done